conference.js

```diff
@@ -2480,6 +2483,14 @@ export default {
      * requested
      */
     hangup(requestFeedback = false) {
+        const state = APP.store.getState();
+
+        // The conference we have already joined or are joining.
+        const conference_ = getCurrentConference(state);
+
+
+        APP.store.dispatch(conferenceWillLeave(conference_));
+        PersistenceRegistry.persistState(APP.store.getState());
```

Comment:
I don't think you should be calling `persistState`. There is a middleware which persist the state on every dispatch of action.